src/content/docs/implementation/setup/content-delivery-network.mdx

```diff
@@ -439,7 +509,7 @@ Then, follow the instructions in [Setup push invalidation for Fastly](https://ww
 
 To validate your CDN setup, use `curl` requests to check expected responses from the paths you have configured.
 
-### Edge content encoding, surrogate key, and cache
+### Content encoding, surrogate key, and cache
 
 Validate that surrogate key, cache hits, and content encoding are working as expected by requesting an Edge Delivery Services-served asset from your Commerce domain. Ensure you are checking a warm cache by making the request at least twice. The following examples are to a warmed cache.
```

Comment:
Please update this script snippet to use https://www.aemshop.net, not just https://aemshop.net.

With the new auto-redirect vcl, you need to send the curl with www.